tools/toolchains/iar.py

```diff
@@ -89,6 +89,10 @@ def __init__(self, target, notify=None, macros=None,
         self.cppc = [main_cc]
         self.cc += self.flags["common"] + c_flags_cmd + self.flags["c"]
         self.cppc += self.flags["common"] + c_flags_cmd + cxx_flags_cmd + self.flags["cxx"]
+        if target.name in ["REALTEK_RTL8195AM"]:
+            self.cc += ["--enum_is_int"]
+            self.cppc += ["--enum_is_int"]
```

Comment:
Do we have any method to skip calling ielftool when using IAR toolchain? We have a different way to generate bin file for IAR toolchain so we want to reduce the time consumption.